curriculum/challenges/spanish/01-responsive-web-design/basic-css/use-rgb-values-to-color-elements.spanish.md

```diff
@@ -8,17 +8,10 @@ localeTitle: Usar valores RGB para elementos de color
 
 
 ## Descripción
+<section id="description"> Otra forma de representar colores en CSS es mediante el uso de valores <code>RGB</code> . El valor RGB para negro se ve así: <code>rgb(0, 0, 0)</code> El valor RGB para blanco se ve así: <code>rgb(255, 255, 255)</code> En lugar de usar seis dígitos hexadecimales como lo hace con el código hexadecimal, con <code>RGB</code> usted debe especificar el brillo de cada color con un número entre 0 y 255. Si hace los cálculos, los dos dígitos de un color son 16 veces 16, lo que nos da 256 valores totales. Entonces, <code>RGB</code> , que comienza a contar desde cero, tiene exactamente el mismo número de valores posibles que el código hexadecimal. Aquí hay un ejemplo de cómo deberías cambiar el fondo del cuerpo a naranja usando su valor en RGB. <blockquote> cuerpo { <br> background-color: rgb (255, 165, 0); <br> } </blockquote></section>
```

Comment:
```suggestion
<section id="description"> Otra forma de representar colores en CSS es mediante el uso de valores <code>RGB</code> . El valor RGB para negro se ve así: <code>rgb(0, 0, 0)</code> El valor RGB para blanco se ve así: <code>rgb(255, 255, 255)</code> En lugar de usar seis dígitos hexadecimales como lo hace con el código hexadecimal, con <code>RGB</code> usted debe especificar el brillo de cada color con un número entre 0 y 255. Si hace los cálculos, los dos dígitos de un color son 16 veces 16, lo que nos da 256 valores totales. Entonces, <code>RGB</code> , que comienza a contar desde cero, tiene exactamente el mismo número de valores posibles que el código hexadecimal. Aquí hay un ejemplo de cómo deberías cambiar el fondo del cuerpo a naranja usando su valor en RGB. 
```css
 body {
   background-color: rgb (255, 165, 0);
 }
```
</section>

Reply:
Didn't realize that block quote was there,, it can be removed, i think there's no blockquote in the english version though. Or at least there's no use for it.